test/plugin/base_test.rb

```diff
@@ -338,6 +338,81 @@ def test_structured_payload_log
     end
   end
 
+  def test_stackdriver_trace_format_compliant_trace
+    full_stackdriver_trace = \
```

Comment:
Constant `FULL_STACKDRIVER_TRACE`?

Reply:
The project we feed into the tests is a constant, though... So we could construct the full expected value as a constant.